apps/bcc3d.ca/pages/donations.html

```diff
@@ -0,0 +1,89 @@
+---
+title: Volunteers — BC Covid-19 3D Printing Group
+description: We're printing crowd-sourced PPE for British Columbia's health care providers and first responders.
+layout: default
+permalink: /donations/
+---
+
+<div class="container">
+  <h2>Donations</h2>
+
+  <div class="row">
+    <div class="col-md-4 order-2">
+      <h5>Donation Links</h5>
+
+      <ul class="list-group mb-4">
+        <li class="list-group-item">
+          <a href="mailto:admin@bcc3d.ca"
+            ><i class="fad fa-envelope fa-fw"></i> Donate Supplies</a
+          >
+        </li>
+        <li class="list-group-item">
+          <a href="mailto:donations@bcc3d.ca"
+            ><i class="fad fa-donate fa-fw"></i> Donate Using E-transfer</a
+          >
+        </li>
+        <li class="list-group-item">
+          <a
+            href="https://www.gofundme.com/f/3d-printed-face-shields-and-ear-savers"
+            ><i class="fad fa-donate fa-fw"></i> Donate on GoFundMe</a
+          >
+        </li>
+      </ul>
+    </div>
+    <div class="col-md-8">
+      <p class="lead">
+        Thanks for your interest in supporting the BC COVID-19 3D Printing
+        Group. Your support is very important to us.
+      </p>
+      <p>
+        Demand for our products is increasing daily as more health care
+        professionals become aware of our group and our products. We are rising
```

Comment:
```suggestion
        professionals run out of traditional personal protective equipment. We are rising
```

Reply:
HCP's don't need to know about us directly now that PHSA will be our distribution channel to the HCP's